examples/hello_compute_c/main.c

```diff
@@ -0,0 +1,139 @@
+#define BIND_GROUP_LAYOUTS_LENGTH (1)
+
+WGPUByteArray read_file(const char *name) {
+    FILE *file = fopen(name, "rb");
+    fseek(file, 0, SEEK_END);
+    long length = ftell(file);
+    unsigned char *bytes = malloc(length);
+    fseek(file, 0, SEEK_SET);
+    fread(bytes, 1, length, file);
+    fclose(file);
+    return (WGPUByteArray){
+        .bytes = bytes,
+        .length = length,
+    };
+}
+
+void read_buffer_map(
+    WGPUBufferMapAsyncStatus status, const uint8_t *data, uint8_t *userdata) {
+    if (status == WGPUBufferMapAsyncStatus_Success) {
+        printf("Times: %s \n", data);
+    }
+}
+
+int main(int argc, char *argv[]) {
+
+    if (argc != 5) {
+        return 0;
+    }
+
+    int numbers[] = {
+        atoi(argv[1]),
+        atoi(argv[2]),
+        atoi(argv[3]),
+        atoi(argv[4]),
+    };
+
+    int size = sizeof(numbers);
+
+    uint32_t numbers_length = sizeof(numbers) / sizeof(int);
+
+    WGPUInstanceId instance = wgpu_create_instance();
+
+    WGPUAdapterId adapter = wgpu_instance_get_adapter(instance,
+        &(WGPUAdapterDescriptor){
+            .power_preference = WGPUPowerPreference_LowPower,
+        });
+
+    WGPUDeviceId device = wgpu_adapter_request_device(adapter,
+        &(WGPUDeviceDescriptor){
+            .extensions =
+                {
+                    .anisotropic_filtering = false,
+                },
+        });
+
+	uint8_t *staging_memory;
+
+    WGPUBufferId staging_buffer = wgpu_device_create_buffer_mapped(device,
+        &(WGPUBufferDescriptor){.size = numbers_length,
+            .usage = WGPUBufferUsage_MAP_READ | WGPUBufferUsage_TRANSFER_DST |
+                WGPUBufferUsage_TRANSFER_SRC},
+            &staging_memory);
+
+    WGPUBufferId storage_buffer = wgpu_device_create_buffer(device,
+        &(WGPUBufferDescriptor){.size = size,
+            .usage = WGPUBufferUsage_STORAGE | WGPUBufferUsage_TRANSFER_DST |
+                WGPUBufferUsage_TRANSFER_SRC});
+
+    WGPUBindGroupLayoutId bind_group_layout =
+        wgpu_device_create_bind_group_layout(device,
+            &(WGPUBindGroupLayoutDescriptor){
+                .bindings = &(WGPUBindGroupLayoutBinding){.binding = 0,
+                    .visibility = WGPUShaderStage_COMPUTE,
+                    .ty = WGPUBindingType_StorageBuffer}});
+
+	WGPUBindingResource resource = (WGPUBindingResource) {
+        .tag = WGPUBindingResource_Buffer,
+        .buffer = (WGPUBindingResource_WGPUBuffer_Body) {
+            storage_buffer
+        }};
+
+    WGPUBindGroupId bind_group = wgpu_device_create_bind_group(device,
+            &(WGPUBindGroupDescriptor){.layout = bind_group_layout,
+                .bindings = &(WGPUBindGroupBinding){
+                    .binding = 0, .resource = resource}});
+
+    WGPUPipelineLayoutId pipeline_layout =
+        wgpu_device_create_pipeline_layout(device,
+            &(WGPUPipelineLayoutDescriptor){
+                .bind_group_layouts = &bind_group_layout});
```

Comment:
Similarly we need to specify `.bind_group_layouts_length` here. There is already a macro for `BIND_GROUP_LAYOUTS_LENGTH`, so we can just use `.bind_group_layouts_length = BIND_GROUP_LAYOUTS_LENGTH`